Source/ServiceCalls/ServiceCaller.swift

```diff
@@ -18,16 +18,24 @@ public enum SerivceCallerSetupError: Error, Equatable {
   case noCallFailedCallback
 }
 
+public struct DataBundleKeys {
+  static let postParameters: String = "parameters"
+}
+
 public class ServiceCaller {
   
   public var callSucceeded: ((Data, DataBundle) -> Void)?
   public var callFailed: ((ServiceCallError) -> Void)?
   
-  public func makeServiceCall(with url: URL, and dataBundle: DataBundle) throws {
+  public func makeServiceCall(with url: URL,
+                              and dataBundle: DataBundle,
+                              usingMethod method: String = "GET") throws {
```

Comment:
Consider using an enum for method like
```
enum method {
case get
case post
}
```

That way you can make method of type enum and when you call it use .get or .post.
This way you are less likely to have issues caused by something like a spelling error or typo